src/main/java/step2/OperatorInterface.java

```diff
@@ -0,0 +1,6 @@
+package step2;
+
+@FunctionalInterface
+public interface OperatorInterface {
+    int apply(int a,int b);

```

Comment:
```suggestion
    int apply(int a, int b);
```

사소한 지적이지만, IDE 의 auto format 기능을 이용하고 계시지 않은듯 한데,
IntelliJ 의 auto format on save 관련 기능들을 검색해서 사용해보시기를 권장드립니다.
저는 개인적으로 [Save Actions](https://plugins.jetbrains.com/plugin/7642-save-actions) 플러그인을 애용합니다.

Reply:
오 감사합니다!